demos/common/cpp/utils/src/performance_metrics.cpp

```diff
@@ -104,6 +104,11 @@ void PerformanceMetrics::logTotal() const {
     slog::info << "\tFPS: " << metrics.fps << slog::endl;
 }
 
+uint32_t PerformanceMetrics::getFrameCount() const {
```

Comment:
What is this for?

Reply:
this is to avoid counting the frames by hands while they are already counted in `PerformanceMetrics`

Reply:
But you still count them by hands in this function

Reply:
@Wovchena Not really. I meant, this function can be used in **demo** instead of counting frames there by creating local variable `int frameCount`

see main.cpp:339
```
if (videoWriter.isOpened() && outputFramesLimit == 0 || **metrics.getFrameCount()** <= outputFramesLimit)) {
    videoWriter.write(image);
}
```

Reply:
From my point of view, it's a small but really convenient feature, and `PerfMetrics` lacks it - I think it can be used in almost all the OMZ demos where frames are being counted, as I said, "by hands" instead of just retrieving already existing info from `Metrics`

But maybe I should've suggested it in a separate PR, with suggesting it in demos where I see it can be helpful

Reply:
I see

Reply:
I have an idea which includes complete hiding counters from `main()`: https://github.com/openvinotoolkit/open_model_zoo/pull/3110

I think you should wait adding `getFrameCount()` func

Reply:
OK, then I'll remove these changes for now, leaving a counter in main()

Reply:
Done